contracts/SortitionPool.sol

```diff
@@ -21,10 +21,17 @@ contract SortitionPool is SortitionTree {
     uint256 minimumStake;
     uint256 poolWeightDivisor;
     address owner;
+    bool locked;
```

Comment:
What if we unwrap all these parameters from `PoolParams` structure? I think in the past we were passing this struct as a parameter to some functions, now it is no longer the case. Additionally, `stakingContract` and `poolWeightDivisor` could be made `immutable` to save on gas.

Reply:
Can we also make `minimumStake` governable by owner?